javascriptv3/example_code/medical-imaging/actions/copy-image-set.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {fileURLToPath} from "url";
+
+// snippet-start:[medical-imaging.JavaScript.imageset.copyImageSetV3]
+import {CopyImageSetCommand} from "@aws-sdk/client-medical-imaging";
+import {medicalImagingClient} from "../libs/medicalImagingClient.js";
+
+/**
+ * @param {string} datastoreId - The ID of the data store.
+ * @param {string} imageSetId - The source image set ID.
+ * @param {string} sourceVersionID - The source version ID.
+ * @param {string} destinationImageSetId - The optional ID of the destination image set.
+ * @param {string} destinationVersionId - The optional version ID of the destination image set.
+ */
+export const copyImageSet = async (datastoreId = "xxxxxxxxxxx", imageSetId = "xxxxxxxxxxxx",
```

Comment:
I usually limit the number of arguments to 3. Beyond that I use an Object `{}`. Not a big deal. Just for future reference.